src/python/espressomd/analyze.pyx

```diff
@@ -560,7 +559,7 @@ class Analysis:
             raise ValueError("type_list_b has to be a list!")
 
         if r_max is None:
-            box_l = make_array_locked(< Vector3d > box_geo.length())
```

Comment:
How was this never caught by anyone?

Reply:
I'm not sure, there should have been a warning about the redundant cast according to the [Cython docs on type casting](https://cython.readthedocs.io/en/stable/src/userguide/language_basics.html#type-casting).